docs/module_guides/indexing/vector_store_index.md

```diff
@@ -0,0 +1,123 @@
+# Using VectorStoreIndex
+
+Vector Stores are a key component of retrieval-augmented generation (RAG) and so you will end up using them in nearly every application you make using LlamaIndex, either directly or indirectly.
+
+Vector stores accept a list of [`Node` objects](/module_guides/loading/documents_and_nodes/root.md) and build an index from them
+
+## Basic usage
```

Comment:
should high-level API, using ingestion pipeline, and creating managing nodes directly explicitly be marked as separate options? E.g. top-level section is `## Loading and Ingesting Data` and underneath there's `### High-Level API`, `### Using Ingestion Pipeline`, and `### Inserting Nodes Directly`

Reply:
Good point; fixed.